src/classes/BDI_DataImportService.cls

```diff
@@ -77,13 +77,13 @@ global with sharing class BDI_DataImportService {
     * @param batchId id of batch to load
     * @return DataImportBatch__c
     */
-    private static DataImportBatch__c loadBatch(ID batchId) {
+    private static DataImportBatch__c loadBatch(Id batchId) {
 
-        List<DataImportBatch__c> listDIBatch = BDI_DataImportBatch_SEL.selectByIds(new Set<Id>{batchId});
+        List<DataImportBatch__c> listDIBatch = BDI_DataImportBatch_SEL.selectByIds(new Set<Id>{ batchId });
```

Comment:
Without knowing how extensive the `listDIBatch` var is used, it's probably better named as `dataImportBatches`.

Reply:
I suspect there are a ton of vars that use the old Hungarian notation (`str*`, `list*`, `set*`, `map*`, etc.). I recommend renaming anything that's local in methods to remove the type prefix. Instance or static vars might be more complex, but I'll leave that up to you.